### PR TITLE
Add alert documentation for OSDCPULoadHigh alert

### PR DESCRIPTION
**Pull request by weirdwiz**
This PR adds the documentation for OSDCPULoadHigh alert

### CODE REVIEW

**Comment by agarwal-mudit** [comment]
@jmolmo PTAL

File: alerts/openshift-container-storage-operator/OSDCPULoadHigh.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+This alert indicates that the CPU usage in the OSD (Object Storage Daemon)
+container on a specific pod has exceeded 80%, potentially affecting the
+performance of the OSD.
+
+## Impact
+
+OSD is a critical component in Ceph storage, responsible for managing data
+ placement and recovery. High CPU usage in the OSD container suggests increased
+ processing demands, potentially leading to degraded storage performance.
+ Creating more OSDs or adding nodes to the cluster can help distribute the load
```
Comment (jmolmo) [inline alerts/openshift-container-storage-operator/OSDCPULoadHigh.md:14]: I would move the paragraph starting with "creating more OSDs.. to the "mitigation" section":
and I would rephrase it as:
Increase the number of OSDs in the cluster adding more new storage devices in the existing nodes or adding new nodes with new storage devices. This would help to distribute ....

File: alerts/openshift-container-storage-operator/OSDCPULoadHigh.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+## Mitigation
+
+If the OSD CPU usage is consistently high, consider taking the following steps:
+
+1. Evaluate the overall storage cluster performance and identify the OSDs
+ contributing to high CPU usage.
+2. Determine if there is a need to add additional OSDs or nodes to the cluster
+ to distribute the load.
+
+### Adding Nodes to the Cluster
```
Comment (jmolmo) [inline alerts/openshift-container-storage-operator/OSDCPULoadHigh.md:37]: I would include also a section for adding new "storage devices" to existing nodes

**Review by jmolmo — CHANGES_REQUESTED** [review]

**Comment by weirdwiz** [comment]
pushed changes, can you ptal @jmolmo ?

**Review by jmolmo — APPROVED** [review]

**Review by agarwal-mudit — APPROVED** [review]
/lgtm
